cmd/uplink/cmd/setup.go

```diff
@@ -71,19 +71,41 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 
 	var override map[string]interface{}
 	if !setupCfg.NonInteractive {
-		_, err = fmt.Print("Enter your Satellite address: ")
+		_, err = fmt.Print(`
+Pick satellite to use:
+  [1] mars.tardigrade.io
+  [2] jupiter.tardigrade.io
+  [3] saturn.tardigrade.io
+Please enter numeric choice or enter satellite address manually [1]: `)
 		if err != nil {
 			return err
 		}
+		satellites := []string{"mars.tardigrade.io", "jupiter.tardigrade.io", "saturn.tardigrade.io"}
 		var satelliteAddress string
-		_, err = fmt.Scanln(&satelliteAddress)
+		n, err := fmt.Scanln(&satelliteAddress)
 		if err != nil {
-			return err
+			if n == 0 {
+				// fmt.Scanln cannot handle empty input
+				satelliteAddress = satellites[0]
+			} else {
+				return err
+			}
 		}
 
 		// TODO add better validation
 		if satelliteAddress == "" {
-			return errs.New("Satellite address cannot be empty")
+			return errs.New("satellite address cannot be empty")
+		} else if len(satelliteAddress) == 1 {
+			switch satelliteAddress {
```

Comment:
nit:  this logic could would be easier to maintain if it were indexed based (if we ever add other satellites)

Reply:
yeah, current code is quite naive :) but I think we don't need anything special for now